types/react-native-animated-progress/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for react-native-animated-progress 1.0
+// Project: https://github.com/kcotias/react-native-animated-progress
+// Definitions by: tup1tsa <https://github.com/tup1tsa>
```

Comment:
You added someone other than yourself? Who is this and do they know they’re being added? 😅

Reply:
It's complicated 😁